darshan-runtime/test/tst_runs.sh

```diff
@@ -0,0 +1,278 @@
+       continue
+    fi
+    # echo "${list[$i]}${list[$j]}${list[$k]}${list[$l]}"
+    OPTS="$OPTS ${list[$i]}${list[$j]}${list[$k]}${list[$l]}"
+done
+done
+done
+done
+
+for ((i=0; i < nelems; i++)); do
+for ((j=i+1; j < nelems; j++)); do
+for ((k=j+1; k < nelems; k++)); do
+for ((l=k+1; l < nelems; l++)); do
+for ((m=l+1; m < nelems; m++)); do
+    clear_opt
+    check_opt ${list[$i]}
+    check_opt ${list[$j]}
+    check_opt ${list[$k]}
+    check_opt ${list[$l]}
+    check_opt ${list[$m]}
+
+    check_skip_opt
+    if test $do_skip = yes ; then
+       continue
+    fi
+    # echo "${list[$i]}${list[$j]}${list[$k]}${list[$l]}${list[$m]}"
+    OPTS="$OPTS ${list[$i]}${list[$j]}${list[$k]}${list[$l]}${list[$m]}"
+done
+done
+done
+done
+done
+
+for ((i=0; i < nelems; i++)); do
+for ((j=i+1; j < nelems; j++)); do
+for ((k=j+1; k < nelems; k++)); do
+for ((l=k+1; l < nelems; l++)); do
+for ((m=l+1; m < nelems; m++)); do
+for ((n=m+1; n < nelems; n++)); do
+    clear_opt
+    check_opt ${list[$i]}
+    check_opt ${list[$j]}
+    check_opt ${list[$k]}
+    check_opt ${list[$l]}
+    check_opt ${list[$m]}
+    check_opt ${list[$n]}
+
+    check_skip_opt
+    if test $do_skip = yes ; then
+       continue
+    fi
+    # echo "${list[$i]}${list[$j]}${list[$k]}${list[$l]}${list[$m]}${list[$n]}"
+    OPTS="$OPTS ${list[$i]}${list[$j]}${list[$k]}${list[$l]}${list[$m]}${list[$n]}"
+done
+done
+done
+done
+done
+done
+
+for ((i=0; i < nelems; i++)); do
+for ((j=i+1; j < nelems; j++)); do
+for ((k=j+1; k < nelems; k++)); do
+for ((l=k+1; l < nelems; l++)); do
+for ((m=l+1; m < nelems; m++)); do
+for ((n=m+1; n < nelems; n++)); do
+for ((o=n+1; o < nelems; o++)); do
+    clear_opt
+    check_opt ${list[$i]}
+    check_opt ${list[$j]}
+    check_opt ${list[$k]}
+    check_opt ${list[$l]}
+    check_opt ${list[$m]}
+    check_opt ${list[$n]}
+    check_opt ${list[$o]}
+
+    check_skip_opt
+    if test $do_skip = yes ; then
+       continue
+    fi
+    # echo "${list[$i]}${list[$j]}${list[$k]}${list[$l]}${list[$m]}${list[$n]}${list[$o]}"
+    OPTS="$OPTS ${list[$i]}${list[$j]}${list[$k]}${list[$l]}${list[$m]}${list[$n]}${list[$o]}"
+done
+done
+done
+done
+done
+done
+done
+
+echo "OPTS=$OPTS"
+
+export LD_PRELOAD=../lib/.libs/libdarshan.so
+
+for exe in ${check_PROGRAMS} ; do
+
+   if test "x$exe" = xtst_mpi_io ; then
+
+      DARSHAN_LOG_FILE="${TST_DARSHAN_LOG_PATH}/${USERNAME_ENV}_${exe}*"
+      echo "DARSHAN_LOG_FILE=$DARSHAN_LOG_FILE"
+
+      DARSGAN_FIELD=MPIIO_BYTES_WRITTEN
+      for opt in "" ${OPTS} ; do
+          if test "x$opt" = x ; then
+             CMD="${TESTMPIRUN} -n ${NP} ./$exe $TEST_FILE"
+          else
+             CMD="${TESTMPIRUN} -n ${NP} ./$exe -$opt $TEST_FILE"
+          fi
+          # echo "CMD=$CMD"
+          rm -f $TEST_FILE $DARSHAN_LOG_FILE
+          $CMD
+          EXPECT_NBYTE=`stat -c %s $TEST_FILE`
+          nbytes=`$DARSGAN_PARSER ${DARSHAN_LOG_FILE} | grep $DARSGAN_FIELD | cut -f5`
```

Comment:
huh.. ok so not a typo? what is this 'DARSGAN' convention?

Reply:
It is a typo. Just the same typo is used in lines below.